cwms-data-api/src/main/java/cwms/cda/data/dto/forecast/ForecastTimeSeries.java

```diff
@@ -0,0 +1,18 @@
+package cwms.cda.data.dto.forecast;
+
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlAttribute;
+import javax.xml.bind.annotation.XmlRootElement;
+
+@XmlRootElement(name = "alias")
+@XmlAccessorType(XmlAccessType.FIELD)
+public class ForecastTimeSeries {

```

Comment:
I'll argue against the actual timeseries object, but agree the codes should be hidden we currently treat those as opaque handles that the user never sees. So generally we expose (office, <further identifying element(s)>). Though it would not be unreasonable to encode that into a "SLUG"/"Key" for applications to use and have an easier time referencing things.

For the timeseries, since that a given forecast spec may have upwards of "we don't actually limit the number" timseries gathering, formatting, and transmitting them all will take too long. Instead a list of URLs to retrieve said timeseries should be provided.

NOTE: for write that's not true so we'll actually still need the time series objects here. Unless we also want to force the downstream users to write to the individual timeseries; which, maybe.

Reply:
I think that muddies up the API necessarily having read/write being such drastically different shapes. I'd argue that the forecast spec (change from forecast instance) would have the list of time series ids. Since we've moved that list away from the instance, a URL is no longer helpful as we don't have the start/end/versionDate to pre-populate the URL parameters with. 

If we think there isn't going to be any extra metadata in the ForecastTimeSeries object, then we should just remove this DTO entirely as the current metadata is only needed on storage and can be set to just query parameters. 

i.e. /forecast/forecast-ts?forecast-spec-id=blah&forecast-date=now would accept and return TimeSeries objects only (if we don't add time window parameters it would default to the forecast instance start/end)

Reply:
True, that's not a good mismatch to have. Just returning the spec/instance and letting that provide the appropriate information seems reasonable. I don't think it unreasonable to add additional support to the time series end point to accept a list of time series, but at the same time and application could just chose to push in parallel.

Reply:
I agree that this DTO is most likely unneeded, I'll remove it for now.

If the Time Series ID list is moving from the Forecast Instance to the Forecast Spec, does that mean Forecast Spec will not have a list of Time Series IDs in itself?